com.wamas.ide.launching/src/com/wamas/ide/launching/validation/LcDslValidator.xtend

```diff
@@ -418,7 +421,7 @@ class LcDslValidator extends AbstractLcDslValidator {
 				error("Test method " + methodName + " does not exist in class " + className, cfg.test, LC.testConfig_Method)
 			} else if (type.methods.stream().filter([method|method.elementName.equals(methodName)]).noneMatch([method | Flags.isPublic(method.flags)])) {
 				error("Test method " + methodName + "  in class " + className + " must be public", cfg.test, LC.testConfig_Method)
-			} else if (type.methods.stream().filter([method|method.elementName.equals(methodName)]).flatMap([method | method.annotations.stream]).noneMatch([annotation | annotation.elementName.equals("Test")])) {
+			} else if (type.methods.stream().filter([method|method.elementName.equals(methodName)]).flatMap([method | method.annotations.stream]).noneMatch([annotation | TEST_ANNOTATION_NAMES.contains(annotation.elementName)])) {
```

Comment:
Would be good to adjust the error message as well.

Reply:
I have included the list in the error message.